lib/Raven/Breadcrumbs/ErrorHandler.php

```diff
@@ -21,16 +21,11 @@ public function __construct(\Raven\Client $ravenClient)
 
     public function handleError($code, $message, $file = '', $line = 0, $context = array())
     {
-        $this->ravenClient->breadcrumbs->record(array(
-            'category' => 'error_reporting',
-            'message' => $message,
-            'level' => $this->ravenClient->translateSeverity($code),
-            'data' => array(
-                'code' => $code,
-                'line' => $line,
-                'file' => $file,
-            ),
-        ));
+        $this->ravenClient->leaveBreadcrumb($this->ravenClient->translateSeverity($code), Breadcrumb::TYPE_ERROR, 'error_reporting', $message, [
```

Comment:
IMO its changing code "just to change it". This means anything instrumenting with breadcrumbs has to change to an entirely new API, and frankly I dont see the reason. A lot of this seems personal preference over being actually better. There's no harm in exposing various APIs to developers. There's a level of trust we ensure, and the breadcrumbs API was considered public, especially since there could exist a world where third parties want to instrument there app using breadcrumbs.

I'm all for improving things, but not changing things drastically (with negative side effects) unless there's tangible benefits.